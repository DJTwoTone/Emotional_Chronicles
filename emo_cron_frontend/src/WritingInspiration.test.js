import React from 'react';
import { render } from '@testing-library/react';
import { MemoryRouter } from 'react-router-dom';
import WritingInspiration from './WritingInspiration';

it('should render', function() {
    render(
        <MemoryRouter>
            <WritingInspiration />
        </MemoryRouter>
    );
})

it('should match the snapshot', function() {
    const { asFragment } = render(
        <MemoryRouter>
            <WritingInspiration />
        </MemoryRouter>
    );
    expect(asFragment()).toMatchSnapshot();
})