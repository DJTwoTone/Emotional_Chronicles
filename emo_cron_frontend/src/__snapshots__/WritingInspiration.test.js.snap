// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`should match the snapshot 1`] = `
<DocumentFragment>
  <div
    class="m-4 shadow card"
  >
    <div
      class="card-body"
    >
      <div
        class="card-header"
      >
        Something to inspire you...
      </div>
      <p
        class="display-4 card-text"
      >
        test inspiration
      </p>
      <button
        class="mb-4 btn btn-dark"
        type="button"
      >
        GET MORE INSPIRATION
      </button>
      <div
        class="card-footer"
      >
        <p
          class="card-text"
        >
          Share some inspiration...
        </p>
        <form
          class="form-inline"
        >
          <input
            class="col-10 form-control"
            name="inspText"
            type="text"
            value=""
          />
          <button
            class="ml-2 btn btn-dark"
            type="submit"
          >
            SHARE
          </button>
        </form>
      </div>
    </div>
  </div>
</DocumentFragment>
`;
